spec/constraints/authenticated_user_spec.rb

```diff
@@ -4,40 +4,30 @@
   describe '#matched?' do
```

Comment:
I'd say this behaviour is well covered by the new request spec you introduced. Delete it?